test/xunit.analyzers.tests/PublicMethodShouldBeMarkedAsTestTests.cs

```diff
@@ -38,6 +39,18 @@ public class PublicMethodShouldBeMarkedAsTestTests
         }
 
         [Fact]
+        public async void DoesNotFindErrorForPublicAbstractMethod()
+        {
+            var diagnostics = await CodeAnalyzerHelper.GetDiagnosticsAsync(analyzer,
+@"public abstract class TestClass {
+    [Xunit.Fact] public void TestMethod() { }
+    public abstract void AbstractTestError();
```

Comment:
Nit: Rename to `AbstractMethod`, since it isn't a test method and it doesn't raise an error.

Reply:
Will do